x/tx/aminojson/any.go

```diff
@@ -0,0 +1,60 @@
+package aminojson
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+)
+
+const (
+	typeUrlName protoreflect.Name = "type_url"
+	valueName   protoreflect.Name = "value"
+)
+
+func (aj AminoJSON) marshalAny(message protoreflect.Message, writer io.Writer) error {
+	fields := message.Descriptor().Fields()
+	typeUrlField := fields.ByName(typeUrlName)
```

Comment:
Getting the typeURL and value fields like this seems a bit untuitive to me, I would do:
```go
any := message.Interface().(*anypb.Any)
any.TypeURL, any.Value...
```

though your code is correct too. Maybe just a matter of preference, I'll let you decide to change or not.

Reply:
Same comment for marshal timestamp and duration